extensions/positron-python/python_files/posit/positron/inspectors.py

```diff
@@ -1206,3 +1250,116 @@ def _get_kind(value: Any) -> str:
         return "other"
     else:
         return "empty"
+
+
+def _get_default_display_value(value: Any, *, level: int = 0) -> tuple[str, bool]:
+    # Try to get the representation of the object.
+    try:
+        display_value = repr(value)
+    except Exception:
+        try:
+            display_value = object.__repr__(value)
+        except Exception:
+            try:
+                display_value = f"<no repr available for {get_qualname(value)}>"
```

Comment:
```suggestion
                display_value = f"<no preview available for {get_qualname(value)}>"
```

if this is potentially user facing, do we expect most people to know what repr means?